godel_msgs/srv/SurfaceBlendingParameters.srv

```diff
@@ -1,8 +1,19 @@
 int32 GET_CURRENT_PARAMETERS=0
 int32 GET_DEFAULT_PARAMETERS=1
+int32 SET_PARAMETERS=2
+int32 SAVE_PARAMETERS=3
+
 int32 action
+
+godel_msgs/RobotScanParameters robot_scan
+godel_msgs/SurfaceDetectionParameters surface_detection
+godel_msgs/BlendingPlanParameters blending_plan
+godel_msgs/ScanPlanParameters scan_plan
+
 ---
+
 bool succeeded
 godel_msgs/RobotScanParameters robot_scan
 godel_msgs/SurfaceDetectionParameters surface_detection
 godel_msgs/BlendingPlanParameters blending_plan
+godel_msgs/ScanPlanParameters scan_plan
```

Comment:
I would like to defer this into a separate issue. I feel it will be much easier to change in a more complete system.